wordpress-mobile/WordPress-Android
# [Blogging Prompts] Track prompts setting tapped

Fixes #17884

**Note: This is being merged bypassing the branch protections because of [instability in BuildKite CI](https://www.buildkitestatus.com/incidents/k9nh8yy0px4q) reporting.**

The build ran successfully:
![image](https://user-images.githubusercontent.com/5091503/216728352-25707faa-78be-49a4-90f5-0fe62cded587.png)


To test:
1. Install Jetpack
2. Select a blogging site
3. Go to `Debug settings`
4. Make sure `BloggingPromptsEnhancementsFeatureConfig` is on
5. Go back to `My Site` > `MENU`
6. Tap on `Site Settings`
7. Tap on `Show prompts`
8. **Verify** `blogging_prompts_settings_show_prompts_tapped` event is triggered
9. **Check** the `enabled` attribute corresponds to the new state of the setting

## Regression Notes
1. Potential unintended areas of impact
Triggering the event when the site settings screen opens.

10. What I did to test those areas of impact (or what existing automated tests I relied on)
Code changes + manual testing.

11. What automated tests I added (or what prevented me from doing so)
Added unit testing for the tracking method.

PR submission checklist:
